src/openapi/transport/auth.js

```diff
@@ -52,7 +58,12 @@ function onTransportError(oldTokenExpiry, result) {
                 'Too many authorization errors occurred for different tokens within a specified timeframe for a specific endpoint',
                 result.url,
             );
-            return;
+            throw {
+                message: 'Auth overload',
+                // obscure the real status code so that the queue does not keep retrying
+                status: 500,

```

Comment:
For people analyzing logs 500 might be confusing. Maybe we can at least indicate in message that this is 'synthetic' error we create on our end in relation to failed authorization? I assume status needs to be the same as we have specific logic to handle various error codes and some of them can force re-tries?

Reply:
yep, its not ideal. fixed it.